src/test/java/com/philips/swcoe/cerberus/cerebellum/codemetrics/java/CodeMetricsVerticalWriterServiceTest.java

```diff
@@ -31,102 +39,133 @@
     @BeforeEach
     public void setupJavaCodeMetricsWithDiff() throws IOException {
         codeMetricsDiffService = new CodeMetricsDiffService(previousPath, currentPath);
-        List<String> classConfig = Files.readAllLines(new File(classConfigPath).toPath(), Charset.defaultCharset());
-        List<String> methodConfig = Files.readAllLines(new File(methodConfigPath).toPath(), Charset.defaultCharset());
-        metricsCSVWriterService = new CodeMetricsVerticalWriterService(classConfig, methodConfig, "CSV");
-        metricsPSVWriterService = new CodeMetricsVerticalWriterService(classConfig, methodConfig, "PSV");
-        metricsMDWriterService = new CodeMetricsVerticalWriterService(classConfig, methodConfig, "MD");
-        metricsHTMLWriterService = new CodeMetricsVerticalWriterService(classConfig, methodConfig, "HTML");
+        List<String> classConfig =
+            Files.readAllLines(new File(classConfigPath).toPath(), Charset.defaultCharset());
+        List<String> methodConfig =
+            Files.readAllLines(new File(methodConfigPath).toPath(), Charset.defaultCharset());
+        metricsCSVWriterService =
+            new CodeMetricsVerticalWriterService(classConfig, methodConfig, "CSV");
+        metricsPSVWriterService =
+            new CodeMetricsVerticalWriterService(classConfig, methodConfig, "PSV");
+        metricsMDWriterService =
+            new CodeMetricsVerticalWriterService(classConfig, methodConfig, "MD");
+        metricsHTMLWriterService =
+            new CodeMetricsVerticalWriterService(classConfig, methodConfig, "HTML");
     }
 
     @Test
     public void shouldWriteClassMetricsandMethodMetricsInVerticalFashion() throws Exception {
-        List<CodeMetricsClassResult> codeMetricsClassResultList = codeMetricsDiffService.getMetricsFromSourceCode();
+        List<CodeMetricsClassResult> codeMetricsClassResultList =
+            codeMetricsDiffService.getMetricsFromSourceCode();
```

Comment:
This line is common in all the tests. Would it be good to pull this into the @BeforeEach method?